lib/litmus_paper/app.rb

```diff
@@ -33,6 +38,60 @@ class App < Sinatra::Base
       _create_status_file(StatusFile.global_health_file)
     end
 
+    get "/:service/status.json" do
```

Comment:
Why did you decide to add new routes rather than changing the response format based on the Accept header in the request, similar to what elasticsearch does, https://www.elastic.co/guide/en/elasticsearch/reference/6.2/cat.html?